cmd/podman/varlink/io.podman.varlink

```diff
@@ -435,6 +435,24 @@ type Runlabel(
     opts: [string]string
 )
 
+# EventInput describes the input to obtain libpod events
+type EventInput(
+    filter: []string,
+    since: string,
+    stream: bool,
+    until: string
+)
+
+# Event describes a libpod struct
+type Event(
+    id: string,
```

Comment:
You can use a `enum` here, which makes it easier for people to see which kinds of events can occur.

Reply:
To clarify, I suggest making an `enum` type like this:
```
type EventType (container, image, foo, baz)
```
And then use it here and for the `filter` property. Everything on the wire stays the same.

Reply:
I don't see how `ID` and `filter` make any sense as enums - ID is the container/pod ID, and makes no sense. Filters are arguable, but as we're sending them now a string seems to make more sense (so we can encode user-requested values to filter against)

Reply:
Sorry I confused the fields (thought `id` is an event id). I meant status.